modules/transport-netty4/src/javaRestTest/java/org/opensearch/rest/Netty4BadRequestIT.java

```diff
@@ -86,7 +86,7 @@ public void testBadRequest() throws IOException {
             () -> client().performRequest(new Request(randomFrom("GET", "POST", "PUT"), path))
         );
         assertThat(e.getResponse().getStatusLine().getStatusCode(), equalTo(BAD_REQUEST.getStatus()));
-        assertThat(e, hasToString(containsString("too_long_frame_exception")));
+        assertThat(e, hasToString(containsString("too_long_http_line_exception")));
```

Comment:
This message changed from `4.1.74.Final -> 4.1.75.Final`.

Details here: https://github.com/netty/netty/commit/3ba2eed16a627371839c63113cdb39df9d6bfe35#diff-864e434ddf7f115156d8497898df1bb48240c9d488d6652b52c437bf9c91fb96